src/main/webapp/app/exercises/text/manage/cluster-statistics/cluster-statistics.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="container-fluid table-responsive">
+    <h4 jhiTranslate="artemisApp.textExercise.clusterStatistics.title"></h4>
+    <table *ngIf="clusters.length > 0" class="table exercise-table guided-tour-exercise-table">
+        <thead>
+            <tr>
+                <th>
+                    <span jhiTranslate="artemisApp.textExercise.clusterStatistics.clusterId">Cluster ID</span>
+                </th>
+                <th>
+                    <span jhiTranslate="artemisApp.textExercise.clusterStatistics.reusedFeedbackRatio">Cluster Reused Feedback Ratio</span>
+                    <fa-icon [icon]="'sort'"></fa-icon>
+                </th>
+                <th>
+                    <span jhiTranslate="artemisApp.textExercise.clusterStatistics.action">Action</span>
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr *ngFor="let cluster of clusters">
+                <td>{{ cluster.clusterId }}</td>
+                <td>
+                    <div style="padding-top: 5px">
```

Comment:
```suggestion
                    <div class="pt-2">
```
Could we use boostrap spacing classes instead, as defined here: https://getbootstrap.com/docs/5.0/utilities/spacing/

I think `pt-2` should be just fine, but maybe you would want to play with the values that work the best.